flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
since you say the licence is GPL can't this be build for aarch64 too?

Reply:
I'm pretty sure electron support for ARM is a bit shaky and I don't have an ARM device to test it on